UI/UserTurn/index.js

```diff
@@ -1,16 +1,40 @@
-import React, { useState } from "react"
+import React, {useEffect, useState} from "react"
 import userTurnStyles from "./userTurn.module.css"
 import {GetAttributesFromCard} from "../../src/utils/helpers"
 import {ChooseYourJamStat} from "../ChooseYourJamStat";
-import { Card } from "../Card"
-import { Button } from "../Button"
+import {Card} from "../Card"
+import {Button} from "../Button"
 
 export function UserTurn({card: {name, cardDescription}, card, slamJams}) {
+
    const [selectedAttribute, setSelectedAttribute] = useState(null)
+   const attributes = GetAttributesFromCard(card);
+
+   const handleNumbers = (e) => {
+      if ([1, 2, 3, 4, 5].includes(parseInt(e.key))) {
+         setSelectedAttribute(attributes[e.key - 1].description)
+      }
+   }
+
+   useEffect(() => {
+      document.addEventListener('keydown', handleNumbers, false);
+      return () => document.removeEventListener('keydown', handleNumbers)
+   }, []);
+
+   const handleEnter = (e) => {
+      if (e.key === 'Enter') {
+         slamJams(selectedAttribute)
+      }
+   }
+
+   useEffect(() => {
```

Comment:
We could just have one `keydown` listener, then branch off to `handleNumbers` or `handleEnter` based on the key press instead? Then we only have to remove one listener too? But this does work ok :)

Reply:
so the enter event listener is added after the `selectedAttribute` has been chosen.

I tried to check if this attribute had a value in a single handler, but it never seems to have the updated values ( presume because the event handler is attached on mount)

Not sure if there is a way to get the current current scoped `selectedAttribute` value?

Reply:
ah never mind then eh! Weird, I thought the `selectedAttribute` should update, maybe if it was outside the scope of `useEffect`? It's not so important I don't think though 👍